python/python/glide/async_commands/core.py

```diff
@@ -392,6 +392,40 @@ async def _cluster_scan(
         type: Optional[ObjectType] = ...,
     ) -> TResult: ...
 
+    async def _reset_connection_password(
+        self, password: Optional[str], re_auth: bool
+    ) -> TResult: ...
+
+    async def reset_connection_password(
+        self, password: Optional[str], re_auth: bool
+    ) -> TOK:
+        """
+        Reset the current connection password with a new password.
+
+        **Note:** This method updates the client's internal password configuration and does
+        not perform password rotation on the server side.
+
+        This method is useful in scenarios where the server password has changed or when
+        utilizing short-lived passwords for enhanced security. It allows the client to
+        update its password to reconnect upon disconnection without the need to recreate
```

Comment:
Should we mention or add a link there where we describe client auto-reconnection feature? It is never mentioned in our docs.

Reply:
It's in the task of the PR, i will add